pandas/tests/internals/test_internals.py

```diff
@@ -311,7 +311,8 @@ def test_delete(self):
     def test_make_block_same_class(self):
         # issue 19431
         block = create_block("M8[ns, US/Eastern]", [3])
-        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+        with pytest.raises(TypeError, match="unexpected keyword"):
```

Comment:
Should we just remove the dtype argument from this test instead of catching this?

Reply:
without the dtpye kwarg its just a smoke test, at which point id rather just remove this test (which id be completely fine with)

Reply:
I think OK then. As is this isn't testing anything pandas related just the stdlib

Reply:
*OK to remove

Reply:
removed